code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -150,6 +150,159 @@
 
 //Spooky special loot
 
+// Beserker armor
+
+#define MAX_BERSERK_CHARGE 100
+#define PROJECTILE_HIT_MULTIPLIER 1.5
+#define DAMAGE_TO_CHARGE_SCALE 0.75
+#define CHARGE_DRAINED_PER_SECOND 5
+#define BERSERK_DAMAGE_REDUCTION 0.6
+#define BERSERK_ATTACK_SPEED_MODIFIER 0.5
+#define BERSERK_COLOUR rgb(149, 10, 10)
```

Comment:
```suggestion
#define BERSERK_COLOUR "#950a0a"
```
I think this would work better, slightly less overhead to not be calculating the RGB each time when its a constant.